src/main/java/com/dmytrohuk/weborganizer/contacts/ContactController.java

```diff
@@ -0,0 +1,35 @@
+package com.dmytrohuk.weborganizer.contacts;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Remove wildcard imports (I've provided a guide how to disable them in the intellij, see training team chat)